wiki/Modding/en.md

```diff
@@ -150,11 +159,21 @@ _See also: [Song Setup](/wiki/Song_Setup)._
 The actual song setup settings does not impact game play, but it is important to give correct names.
 The artist name, title of the song, source (if applicable), and tags should all be the same in each difficulty.
 
-Audio lead-in times are usually the same, but can differ depending on when the first note is played.
-The `Enable Countdown` checkbox should be either enabled or disabled for all difficulties.
+Here are some other things too look out for in the song setup:
+
+- The `Enable Countdown` checkbox should be either enabled or disabled for all difficulties.
+- The combo colours should be the same on the same background of each a mapset.
```

Comment:
The same thing is better worded in the next bullet point, just delete this line.